### PR TITLE
CP-24409: No need to check vm status when destroy VUSB

### PR DESCRIPTION
**Pull request by taoyongd**
Signed-off-by: Taoyong Ding <taoyong.ding@citrix.com>

### CODE REVIEW

**Comment by taoyongd** [comment]
Thanks @lindig ， Will this able to merge?

File: ocaml/xapi/xapi_vusb.ml [hunk 1]
```diff
@@ -53,7 +53,7 @@ let destroy ~__context ~self =
   let r = Db.VUSB.get_record_internal ~__context ~self in
   let vm = r.Db_actions.vUSB_VM in
   (* Force the user to unplug first *)
-  if Helpers.is_running ~__context ~self:vm  && r.Db_actions.vUSB_attached <> Ref.null then
+  if r.Db_actions.vUSB_attached <> Ref.null then
```
Comment (jonludlam) [inline ocaml/xapi/xapi_vusb.ml]: We usually use `is_valid_ref` rather than a direct comparison with Ref.null

Comment (taoyongd) [reply ocaml/xapi/xapi_vusb.ml]: updated @jonludlam

File: ocaml/xapi/xapi_vusb.ml [hunk 2]
```diff
@@ -53,7 +53,7 @@ let destroy ~__context ~self =
   let r = Db.VUSB.get_record_internal ~__context ~self in
   let vm = r.Db_actions.vUSB_VM in
   (* Force the user to unplug first *)
-  if Helpers.is_running ~__context ~self:vm  && r.Db_actions.vUSB_attached <> Ref.null then
+  if not (Db.is_valid_ref __context r.Db_actions.vUSB_attached) then
```
Comment (jonludlam) [inline ocaml/xapi/xapi_vusb.ml]: I think the `not` is incorrect.

Comment (taoyongd) [reply ocaml/xapi/xapi_vusb.ml]: updated.

**Pull request merged**